### ISSUE TITLE
Add recording of yotta query parameters for future taggability

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/phetcommon#66]
Proposal is to send back query parameters (and if available, their values) that starts with `yotta`.

**Comment by jonathanolson** [comment]
In production, closing.

**Comment by zepumph** [comment]
@jonathanolson can you please describe this feature? I don't know how to use it, or where it is documented. We want to be able to provide custom tags for phet-io sim runs from phet-io clients and PhET Studio presets. Can you point me in a direction?

**Comment by kathy-phet** [comment]
@zepumph @mattpen - I believe this code: 

>  // Forward yotta-specific query parameters, see https://github.com/phetsims/phetcommon/issues/66
    for ( const [ key, value ] of new URLSearchParams( window.location.search ) ) {
      if ( key.startsWith( 'yotta' ) ) {
        pingParams += `&${encodeURIComponent( key )}=${encodeURIComponent( value )}`;
      }
    }
  
  Means that adding any query parameter that has yotta in it will be forwarded to yotta: 
  ?yotta_studio_preset="preset URL" (this would have partner name and preset value in it, and could be parsed further)
  
  @jonathanolson - is this correct? any query parameter that starts with "yotta" are then included and passed with the yotta message

**Comment by jonathanolson** [comment]
> is this correct? any query parameter that starts with "yotta" are then included and passed with the yotta message

Yes, they are passed and recorded. There isn't any code to USE those things yet, but they will be recorded and can be processed (historically AND going forward) whenever we'd add code to process it.
